src/inputs/SelectField.stories.tsx

```diff
@@ -248,6 +248,45 @@ export function PerfTest() {
 }
 PerfTest.parameters = { chromatic: { disableSnapshot: true } };
 
+export function LazyLoadStateFields() {
+  const [selectedValue, setSelectedValue] = useState<string | undefined>(loadTestOptions[2].id);
+  return (
+    <>
+      <SelectField
+        label="Project"
+        value={selectedValue}
+        onSelect={setSelectedValue}
+        unsetLabel={"-"}
+        options={{
+          initial: [loadTestOptions.find((o) => o.id === selectedValue)!],
+          load: async () => {
+            return new Promise((resolve) => {
+              // @ts-ignore - believes `options` should be of type `never[]`
+              setTimeout(() => resolve({ options: loadTestOptions }), 1500);
+            });
+          },
+        }}
+      />
+      <SelectField
+        label="Project 2 (i.e In a SuperDrawer)"
+        value={selectedValue}
+        onSelect={setSelectedValue}
+        unsetLabel={"-"}
+        options={{
+          initial: [loadTestOptions.find((o) => o.id === selectedValue)!],
+          load: async () => {
+            return new Promise((resolve) => {
+              // @ts-ignore - believes `options` should be of type `never[]`
+              setTimeout(() => resolve({ options: loadTestOptions }), 1500);
+            });
+          },
+        }}
+      />
+    </>
+  );
+}
+PerfTest.parameters = { chromatic: { disableSnapshot: true } };
```

Comment:
**nit/request:** This looks like a copy & paste mistake. Can you update this to be the `LazyLoadStateFields.parameters`?

Reply:
Exactly, fixed on last commit.